locales/en-human.json

```diff
@@ -1578,15 +1578,11 @@
       "BUY": "@:BUY"
     },
     "can_sell_max": {
-      "EXPLAIN": "You can sell up to <span class='pointer'>{{::max}} {{::curr}}</span> of your balance.",
+      "EXPLAIN": "You can sell up to <span class='pointer'>{{::max}} {{::curr}}</span>.",
       "KYC": "Increase your limit."
     },
     "can_sell_remaining_balance": {
-      "EXPLAIN": "You can sell up to <span class='pointer'>{{::max}} {{::curr}}</span> of your balance."
-    },
-    "max_remaining_below_min": {
-      "EXPLAIN": "🙁 You have reached your sell limit.",
-      "KYC": "Increase your limit."
+      "EXPLAIN": "You can sell up to <span class='pointer'>{{::max}} {{::curr}}</span>"
```

Comment:
Did you intend to change these sell translations?

Reply:
nope, will fix